doc/misc/filtering.7.rst

```diff
@@ -28,42 +25,49 @@
 
-Exclude Filtering
-=================
+Global excludes filtering
+=========================
 
-Exclude Filtering is a mechanism used by a user or by the ``DNF5`` plugin to modify the set of available
-packages. Exclude Filtering can be modified by either ``includepkgs`` or ``excludepkgs`` configuration options in
-configuration files. In addition to user-configured excludes, plugins can also extend the set of excluded packages.
-To disable excludes from the ``DNF5`` plugin you can use the ``--disable-plugin`` command line option.
+Exclude filtering can be adjusted using the ``includepkgs`` or ``excludepkgs`` configuration options in the DNF5 configuration file. To disable excludes, you can use the ``disable_excludes`` configuration option:
 
-To disable all excludes for e.g. the install command you can use the following combination
-of command line options:
+``dnf5 --setopt=disable_excludes=* install bash``
 
-``dnf5 --disableexcludes=all --disable-plugin="*" install bash``
+For details about the configuration options see :manpage:`dnf5.conf(5)`, :ref:`DNF5 configuration reference <dnf5_conf-label>`.
 
 
-Modular Filtering
-=================
+Repository excludes filtering
+=============================
 
-Please see `the modularity documentation` for details on how Modular Filtering works.
+Similar to global excludes, but this configuration is repository-specific and only affects packages within the repository where the excludes are set.
 
-With modularity, only RPM packages from ``active`` module streams are included in the available
-package set. RPM packages from ``inactive`` module streams, as well as non-modular packages with
-the same name or provides as a package from an ``active`` module stream, are filtered out. Modular
-filtering is not applied to packages added from the command line, installed packages, or packages
-from repositories with ``module_hotfixes=true`` in their ``.repo`` file.
 
-Disabling of modular filtering is not recommended, because it could cause the system to get into
-a broken state. To disable modular filtering for a particular repository, specify
-``module_hotfixes=true`` in the ``.repo`` file or use ``--setopt=<repo_id>.module_hotfixes=true``.
+User excludes filtering
+=======================
+
+API users have an additional option how to fine-tune excluded packages using ``*_user_excludes()`` and ``*_user_includes`` methods of the ``PackageSack`` object. See :ref:`PackageSack API <api_packagesack-label>`.
+
+
+Versionlock
+===========
+
+Additionally, the versionlock functionality is implemented using excludes filtering. However, these excludes are applied only during transactional operations.
+For details see :manpage:`dnf5-versionlock(8)`, :ref:`Versionlock command <versionlock_command_ref-label>`.
+
+
+Modular filtering
+=================
+
+Please see :ref:`the modularity documentation <modularity_misc_ref-label>` for details on how Modular Filtering works.
```

Comment:
Now, when modularity(7) page is packaged, shouldn't this link mention both the manual page and the modularity_misc_ref-label. Similar to one in Versionlock section above?

Reply:
Yeah, we should reference other docs in uniform way. Fixed.